src/frontends/pytorch/src/utils.hpp

```diff
@@ -28,6 +28,7 @@ const std::map<int, element::Type> TORCH_TO_OV_TYPE{
     {5, element::f16},
     {6, element::f32},
     {7, element::f64},
+    {11, element::boolean}
```

Comment:
There is a codestyle warning, I see it is not your changes are guilty in that, but could you please fix it?

Reply:
done